huuugeee name patch!

Yzzuwyzzuf opened the pull request:
## About The Pull Request

changes made:
added pelt prefixes
added color suffixes
added tortie pelt prefixes
added eye suffixes
added prefixes
added suffixes
added loner names
rearranged prefixes and suffixes into different categories

## Why This Is Good For ClanGen

I believe these changes improve the game by adding more consistency between names and appearance. In base game, a black tabby with green eyes could be named Leopardsnow or a  sienna ticked with blue eyes could be named Tigerbee but with these new categories, those cats can have more fitting names like Ravensprout or Foxspeckle. below I will provide reasonings not only for prefixes and suffixes that I feel could use an extra explanation but also I plan on going in depth on the loner names which are listed below the existing names for normal prefixes, suffixes, and loner names and separated into categories for organization and explanation purposes.

suffixes and prefixes

prefixes
squeak prefix - if we have the sounds birds say as a prefix, I believe squeak should be on the table as well. besides, kits do squeak
stray/lost- cats can have a stray/missing thing on them whether its a whisker or tuft of fur or just being lost and astray

Ripple, prefix and suffixes moved to striped pelt slots-every warrior given the ripple prefix is striped. I always imagine ripple as ripple stripes and it makes less sense on say a smoke kitty or a singlestripe kitty
Spot, leopard, and similar prefixes and suffixes moved or added to spotted pelts as every cat with it is spotted
bramble, tiger and similar prefix and suffixes- moved or added to striped pelt slots for obvious reasons
perch prefix- 3 cats have existed with prefix plus perch, like many other prefixes, is prey and its also an action like many other prefixes
Maggot- Maggottail from The Forgotten warrior plus it is a bug they would come across easy so it makes sense that it'd be in the naming repitoire
Rot/dead/mange/wilted/all the more morbid-ish ones - Normal things in their world. food rots, prey dies, fur can become mangy/ a cat can be dull or mangled/scarred. its definitely more shadowclan-esque names but they would easily fit in the world
Mint- 4 cats have existed with mint prefix, catmint is a common herb, plus its cute!
All ice/snow related prefix and suffixes- moved to either white pelts or blue/blue adjacent eyes
Tawny moved to yellow, orange, and brown pelts due to definition of tawny being yellowy brown
Hen- if we're stealing chickens from twolegs, then the cats surely know what hens are plus its a cute prefix
Tortoise- a tortoise and a turtle are different so they should both be on the list as tortoises are the ones that cant swim but turtles can
plump/thin/round- descriptive prefixes

suffixes
Eater- Bugeater from Greystripe's Vow also if we have the suffix of runner, I believe eater and also fisher which I added are equally good suffixes especially paired with prey prefixes
suckle- my first thought was putting it yellow and white for honeysuckle but kits also suckle on their parents 
twitch/stretch/tear/snore/nuzzle- actions cats can do that could add a bit of personality to the cat
stomp- we have step, we have foot, it only makes sense to add stomp. a cat can easily stomp their feet and the name fits insanely well as a suffix
scruff- cats have scruffs they usually are carried by as a kit so going with the body part suffixes, scruff works well
ridge- for one, a spine fits the visual of a ridge quite well and also cats encounter ridges plenty whether its mountain cats or at the ridge of cliff and even hills can have ridges
sprig/herb- a suffix especially fitting for medicine cats, they handle sprigs of herbs quite often 


many other prefixes and suffixes moved or added to other categories due to better color/pelt fits

Loner names
I've organized them by categories and in those categories, I've put the names that are most cat-fitting at the front though I definitely got a little lazy at the end

520: the cat daddy himself Jackson Galaxy! mans the cat whisperer and even after his shows ended he's still spreading valuable cat info and help! if any name deserves adding its this one at the very least!
521: famous cats on social and other forms of media
522: warriors reference names
523: most common names for cats in other countries. I feel this is an especially good add for inclusion purposes and if someone has a cat with one of these common names in their/ a relatives country, it could add a greater attachment 
524: seasons
525: Greek mythological beings 
526: mythological creatures, monsters, and cryptids
527: Pokemon. quite a few good mons like Persian, Purugly, Absol, the eeveelutions, and others were just completely left out! I also lump acerola in as an honorary cat cause of that cat mouth
528: fire emblem 3 houses
529:mario characters since we have Mario in the list already
530: final fantasy. and I'd like to especially point out moogles look like cute little white kits and just sayin
531: elder scrolls, mainly for khajiit but it being a well known and well played game 
532: baldurs gate/dnd
533: fnaf
534: loz/botw
535: kingdom hearts
536: other assorted games I didn't think warranted individual categories
537: movies and shows. aslan and Cheshire especially, they're super well known cats!
538: well loved streamers and YouTubers plus some twitch lingo cause pog
539: horror villains
 540: names a loner or rogue might name them self 
541: item names. could fit into several categories but sits here 
542: human names 
543: names people might name their cats
544: meme/funny names/no other fitting categories
545: names that are more special to me like names of pets, characters, and mt name in the server, fuzzy!

## Proof of Testing

my testing has been done through my mod on the discord, Fuzzy's text expansion mod, the categories work perfectly!
![shade](https://github.com/user-attachments/assets/6a2a8427-b393-481b-b120-0f6b4728103f)
![wilt](https://github.com/user-attachments/assets/bf7865f3-13e4-4cf8-b2a8-36dd3c874aec)
![seed](https://github.com/user-attachments/assets/0b59481a-e1b8-46fe-b16b-2cd54e75f39f)
![frost](https://github.com/user-attachments/assets/d0119e94-6c82-40ec-98ba-e94be041386e)
![raven](https://github.com/user-attachments/assets/843ffcee-33e7-4009-8bbf-e65fcd0213a6)
![straywhisker](https://github.com/user-attachments/assets/8086aa8c-260d-42da-9942-88a740bfff99)

## Changelog/Credits

naming categories added, 
prefixes and suffixes rearranged, 
loner names added
credit Yzzuwyzzuf

Yzzuwyzzuf committed:
Add files via upload

Yzzuwyzzuf committed:
Update names.json

added eye suffixes, color suffixes, pelt prefixes
reorganized names
added lots of loner names and warrior names in all categories

Yzzuwyzzuf committed:
Update names.json

forgot 2 quotation marks

Yzzuwyzzuf committed:
Update names.json

fixing forgotten commas

Yzzuwyzzuf committed:
Update names.json

reverted back to main file names doc

Yzzuwyzzuf committed:
Merge branch 'ClanGenOfficial:development' into development

Yzzuwyzzuf committed:
big name update!

## About The Pull Request

changes made:
added pelt prefixes
added color suffixes
added eye suffixes
added prefixes
added suffixes
added loner names
rearranged prefixes and suffixes into different categories

## Why This Is Good For ClanGen

I believe these changes improve the game by adding more consistency between names and appearance. In base game, a black tabby with green eyes could be named Leopardsnow or a  sienna ticked with blue eyes could be named Tigerbee but with these new categories, those cats can have more fitting names like Ravensprout or Foxspeckle. below I will provide reasonings not only for prefixes and suffixes that I feel could use an extra explanation but also I plan on going in depth on the loner names which are listed below the existing names for normal prefixes, suffixes, and loner names and separated into categories for organization and explanation purposes.

suffixes and prefixes

Eater suffix- Bugeater from Greystripe's Vow
Ripple, prefix and suffixes moved to striped pelt slots-every warrior given the ripple prefix is striped
Spot, leopard, and similar prefixes and suffixes moved or added to spotted pelts as every cat with it is spotted
bramble, tiger and similar prefix and suffixes- moved or added to striped pelt slots for obvious reasons
perch prefix- 3 cats have existed with prefix
Maggot prefix- Maggottail from The Forgotten warrior
Mint prefix- 4 cats have existed with mint prefix
Wild prefix-2 cats have existed with this prefix
All ice/snow related prefix and suffixes- moved to either white pelts or blue/blue adjacent eyes
Tawny moved to yellow, orange, and brown pelts due to definition of tawny
Hen prefix- if we're stealing chickens from twolegs, then the cats surely know what hens are
most other prefixes and suffixes moved or added to other categories due to  better color/pelt fits

Loner names
I've organized them by categories and in those categories, I've put the names that are most cat-fitting at the front though I definitely got a little lazy at the end

520: the cat daddy himself Jackson Galaxy! mans the cat whisperer and even after his shows ended he's still spreading valuable cat info and help! if any name deserves adding its this one at the very least!
521: famous cats on social and other forms of media
522: warriors reference names
523: most common names for cats in other countries. I feel this is an especially good add for inclusion purposes and if someone has a cat with one of these common names in their/ a relatives country, it could add a greater attachment 
524: seasons
525: Greek mythological beings 
526: mythological creatures, monsters, and cryptids
527: Pokemon. quite a few good mons like Persian, Purugly, Absol, the eeveelutions, and others were just completely left out! I also lump acerola in as an honorary cat cause of that cat mouth
528: fire emblem 3 houses
529:mario characters since we have Mario in the list already
530: final fantasy. and I'd like to especially point out moogles look like cute little white kits and just sayin
531: elder scrolls, mainly for khajiit but it being a well known and well played game 
532: baldurs gate/dnd
533: fnaf
534: loz/botw
535: kingdom hearts
536: other assorted games I didn't think warranted individual categories
537: movies and shows. aslan and Cheshire especially, they're super well known cats!
538: well loved streamers and YouTubers plus some twitch lingo cause pog
539: horror villains
 540: names a loner or rogue might name them self 
541: item names. could fit into several categories but sits here 
542: human names 
543: names people might name their cats
544: meme/funny names/no other fitting categories
545: names that are more special to me like names of pets, characters, and mt name in the server, fuzzy!

## Proof of Testing

my testing has been done through my mod on the discord, Fuzzy's text expansion mod, the categories work perfectly!

## Changelog/Credits

naming categories added, prefixes and suffixes rearranged, loner names added

Yzzuwyzzuf committed:
Update names.json

added Tortoise prefix, added Figaro loner name, fixed a coma issue

Yzzuwyzzuf committed:
Update names.json

commas

Yzzuwyzzuf committed:
Delete seasonalthoughts.txt

removed something not relating to the update

DeadGirlRisen commented:
Hi! I took a quick look, and noticed the inappropriate name molesting was removed, can you please re-add it? (I'm just a fellow dev being curious over the pr and noticed this ^^)

Yzzuwyzzuf commented:
> Hi! I took a quick look, and noticed the inappropriate name molesting was removed, can you please re-add it? (I'm just a fellow dev being curious over the pr and noticed this ^^)

Omg I didn't realize how bad that name was until reading your reply yes of course I will!

Yzzuwyzzuf committed:
Update names.json

Readded molesting to inappropriate names

Yzzuwyzzuf committed:
Update names.json

added skunk prefix to white, pale grey, ghost, and black as they are black and white so a cat likely would name their kits accordingly

archanyhm commented:
As outlined by scribble in #2641, names go through rigorous discussion before they're being added. Additionally, some of the loner names that have been added are inappropriate and not to debate for inclusion. Refer to our Discord if you wish to discuss name changes. Closing this for now.
